elements/barcode_128.go

```diff
@@ -43,6 +43,7 @@ type Barcode128 struct {
 
 type Barcode128WithData struct {
 	Barcode128
+	Width    int
```

Comment:
Maybe we can have something like this instead?
```suggestion
	Dimensions  BarcodeDimensions
```
?

Reply:
I think it can be a bit misleading, since `Barcode128` stores `Width` 🤔

Reply:
Ok, fine, then it can be the way it is right now